tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -826,6 +828,72 @@ class WskBasicUsageTests
                 s"(?i)action /${namespace}/${actNameNoDescOrParams}\\s*\\(parameters: none defined\\)")
     }
 
+    it should "save action code to file" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "saveAction"
+            val seqName = "seqName"
+            val dockerName = "dockerName"
+            val containerName = s"bogus${Random.alphanumeric.take(16).mkString.toLowerCase}"
+            val saveName = s"save-as-$name.js"
+            val currentDir = System.getProperty("user.dir")
```

Comment:
temp dir instead?

Reply:
The dir used is based on what directory the CLI is be executed from.

Reply:
Any idea how to change the directory based on the above comment?